catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -351,7 +351,9 @@
                             !org.abego.treelayout.*,
                             *
                         </Import-Package>
-                        <Export-Package/>
+                        <Export-Package>
+                            org.codice.ddf.catalog.ui.metacard.workspace.transformer
```

Comment:
❗️ I do not see this in our current whitelist documentation. Are we expecting to maintain this package going forward? Based on the description it seems like we would. If so, it seems like it should be documented what exactly in that package is being whitelisted: http://artifacts.codice.org/service/local/repositories/snapshots/content/ddf/docs/2.12.0-SNAPSHOT/docs-2.12.0-20180525.142320-200-Documentation.html#_application_whitelists

Reply:
The interface is experimental and should not be whitelisted. It should be maintained going forward, but may change in the process.

Reply:
Okay thanks for the explanation that part sounds good to me. I missed the `experimental` part in `WorkspaceTransformation`.